extra/nrnivmodl_core_makefile.in

```diff
@@ -80,27 +114,41 @@ $(special): $(coremech_lib)
 	  -L $(OUTPUT) -l$(coremech_libname) $(CORENRNLIB_FLAGS) -Wl,-rpath,'$(DESTDIR_RPATH)' -Wl,-rpath,$(libdir) $(LDFLAGS)
 
 
-$(coremech_lib): $(mod_func_o) $(dimplic_o) $(mod_objs) build_always
+$(coremech_lib): $(mod_func_o) $(dimplic_o) $(mod_optmods_objs) $(mod_ispc_objs) build_always
 	@printf " => $(C_GREEN)LINKING$(C_RESET) library $(coremech_lib) Mod files: $(mod_files) (+ $(OPTMODS))\n"
 	$(CXX_LINK_SHARED) -I $(incdir) -DADDITIONAL_MECHS $(datadir)/enginemech.cpp -o ${coremech_lib} ${_SONAME} \
-	  $(mod_func_o) $(dimplic_o) $(mod_objs) $(datadir)/libscopmath.a $(CORENRNLIB_FLAGS) -Wl,-rpath,$(libdir) $(LDFLAGS)
+	  $(mod_func_o) $(dimplic_o) $(mod_optmods_objs) $(mod_ispc_objs) $(datadir)/libscopmath.a $(CORENRNLIB_FLAGS) -Wl,-rpath,$(libdir) $(LDFLAGS)
 
 
-# Generic build cpp->o. Need PIC for shared lib
+# Generic build cpp->.o Need PIC for shared lib
 $(OBJS_DIR)/%.o: $(MODC_DIR)/%.cpp $(kinderiv_h) | $(OBJS_DIR)
 	@printf " -> $(C_GREEN)Compiling$(C_RESET) $<\n"
 	$(CXXCOMPILE) @CMAKE_CXX_COMPILE_OPTIONS_PIC@ -c $< -o $@
 
+# Generic build ispc->.obj Need PIC for shared lib
+$(OBJS_DIR)/%.obj: $(MODC_DIR)/%.ispc | $(OBJS_DIR)
+	@printf " -> $(C_GREEN)Compiling with ISPC$(C_RESET) $<\n"
+	$(ISPC_COMPILE) @CMAKE_ISPC_COMPILE_OPTIONS_PIC@ $< -o $@
+
+# Build cpp and ispc files with mod2c/nmodl
+$(mod_ispc_objects): $(MODC_DIR)/%.ispc: $(MODS_PATH)/%.mod | $(MODC_DIR)
+	@printf " -> $(C_GREEN)MOD2C$(C_RESET) $<\n"
+	MODLUNIT=$(datadir_mod2c)/nrnunits.lib \
+	  @mod2c_install_bin@ $< -o $(MODC_DIR)/ @nmodl_arguments@
+
+$(mod_ispc_c_objects): $(MODC_DIR)/%.cpp: $(MODC_DIR)/%.ispc
```

Comment:
this needs at least a comment so we know in the future what this is for (just an empty rule without any explanation.. for sure someone will think this was a mistake)